packages/frontapp/src/app/components/RadioButton/index.tsx

```diff
@@ -100,6 +101,7 @@ interface IRadioGroup {
   name: string;
   options: { name: string; value: string }[];
   onChange?: (value: string) => void;
+  reset?: boolean;
```

Comment:
if it is boolean, better name it with a question, like `isReset` or 'shouldReset'